tutorials/mongoose/query-string.md

```diff
@@ -0,0 +1,25 @@
+Mongoose's `Model.find()` function is an important method to understand.
+You may call it without any arguments and it will return all the documents in that model.
+You may hone the search by passing in a `filter` which tells mongoose what to look for in the database.
+This `filter` may be an `objectId` or an `object`.
+When using `Model.find()`, you should explicitly list the parameters you are searching for in the model.
+`Model.find()`'s default behavior is that it will return all documents in the model, so if none of the properties
```

Comment:
Break this out into a separate section that demonstrates what happens if `req.query` is empty and you try to do `Model.find(req.query)`